02 - JS and CSS Clock/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>JS + CSS Clock</title>
+  <script src="https://unpkg.com/vue"></script>
+</head>
+
+<body>
+  <div class="clock" id="app">
+    <div class="clock-face">
+      <div class="hand hour-hand"></div>
```

Comment:
Maybe try to assign the style property directly using as the value, the computed property that depends on the now reactive key;

See
```
<div :style="{ fontSize: postFontSize + 'em' }">
   
  </div>
```